twilight-cache-inmemory/src/model/presence.rs

```diff
@@ -14,7 +14,7 @@ use twilight_model::{
 pub struct CachedPresence {
     pub(crate) activities: Vec<Activity>,
     pub(crate) client_status: ClientStatus,
-    pub(crate) guild_id: Id<GuildMarker>,
+    pub(crate) guild_id: Option<Id<GuildMarker>>,
```

Comment:
Actually I do not see a reason for this change. Except that the `from_model` would need to require another arg for the `guild_id`. But since the `cache_presence` method requires a `guild_id` anyways, this shouldn't be a huge problem, no?